includes/apple-exporter/components/class-body.php

```diff
@@ -58,6 +58,75 @@ public static function node_matches( $node ) {
 	}
 
 	/**
+	 * Register all specs for the component.
+	 *
+	 * @param string $text
+	 * @access protected
+	 */
+	public function register_specs() {
+		$this->register_spec(
+			'json',
+			__( 'JSON', 'apple-news' ),
+			array(
+				'role'   => 'body',
+				'text'   => '%%text%%',
+				'format' => '%%format%%',
+			)
+		);
+
+		$this->register_spec(
+			'body-layout',
+			__( 'Layout', 'apple-news' ),
+			array(
+				'columnStart' => '%%body_offset%%',
+				'columnSpan'  => '%%body_column_span%%',
+				'margin'      => array(
+					'top' => 12,
+					'bottom' => 12
+				),
+			)
+		);
+
+		$this->register_spec(
+			'body-layout-last',
+			__( 'Layout for Last Component', 'apple-news' ),
+			array(
+				'columnStart' => '%%body_offset%%',
+				'columnSpan'  => '%%body_column_span%%',
+				'margin'      => array(
+					'top' => 12,
+					'bottom' => 30
+				),
+			)
+		);
+
+		$this->register_spec(
+			'default-body',
+			__( 'Default Style', 'apple-news' ),
+			$this->get_default_style_spec()
+		);
+
+		$this->register_spec(
+			'dropcapBodyStyle',
+			__( 'Dropcap Style', 'apple-news' ),
+			array_merge(
+				$this->get_default_style_spec(),
+				array(
+					'dropCapStyle' => array (
+						'numberOfLines' 			=> '%%numberOfLines%%',
```

Comment:
The token is really just for display purposes. I'll change it but it actually won't affect functionality.